transformer.ts

```diff
@@ -72,7 +72,7 @@ function visitImportClause (node: ts.ImportDeclaration, program: ts.Program): ts
   }
 
   const importSymbol = program.getTypeChecker().getSymbolAtLocation(node.moduleSpecifier)
-  if (!importSymbol || !isOurStubModule(importSymbol.valueDeclaration.getSourceFile())) {
+  if (!importSymbol || !isOurStubModule(importSymbol.valueDeclaration!.getSourceFile())) {
```

Comment:
This will fail in runtime if `valueDeclaration` is undefined for some reason.

Reply:
Yes.
It was previously optional, but isn't any longer. Happy to make it more robust, but given that it wouldn't introduce a new error (just continue to fail on a previously existing one), wasn't sure if you would rather this or to introduce nullcheck logic.

happy either way